openmc/data/multipole.py

```diff
@@ -112,7 +112,8 @@ def _broaden_wmp_polynomials(E, dopp, n):
         erf_beta = 1.0
         exp_m_beta2 = 0.0
     else:
-        erf_beta = np.erf(beta)
+        from scipy.special import erf
```

Comment:
In my opinion, this import shouldn't be nested in the middle of the function like this.